src/viur/core/bones/relational.py

```diff
@@ -1098,7 +1098,10 @@ def get_values(skel_, values_cache):
 
         return result
 
-    def createRelSkelFromKey(self, key: t.Union[str, "db.Key"], rel: dict | None = None):
+    def createRelSkelFromKey(self, key: db.Key, rel: dict | None):
+        return self.createRelSkelFromKey([key, rel])
```

Comment:
This cannot function!
```suggestion
        return self.createRelSkelFromKeys([key, rel])
```